src/components/core/DpUpload/DpUploadFiles.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta } from '@storybook/addon-docs'
+import DpUploadFiles from './DpUploadFiles'
+
+<Meta
+    title="Components/UploadFiles"
+    component={DpUploadFiles}
+/>
+
+# UploadFiles
+
+The UploadFiles component enables users to upload files by dropping them onto a designated area or by triggering the process through a button activation.
+
+## Custom usage
+
+```html
+<dp-upload-files
```

Comment:
I'd suggest to only add attrs here which are required.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/518/commits/9ec9e27c8b9f8408f9ff481f42a532bc858117bf

Reply:
https://github.com/demos-europe/demosplan-ui/pull/518/commits/071b8c4f9a8e0498d8f3f290dcdc209075f19ff0